build/packages.targets

```diff
@@ -17,51 +18,59 @@
         <PackageReference Update="Microsoft.VisualStudio.Setup.Configuration.Interop" Version="1.16.30" />
         <PackageReference Update="Microsoft.VisualStudio.Shell.15.0" Version="$(VSFrameworkVersion)" />
-        <PackageReference Update="Microsoft.VisualStudio.Shell.Embeddable" Version="$(VSFrameworkVersion)" />
         <PackageReference Update="Microsoft.VisualStudio.Shell.Framework" Version="$(VSFrameworkVersion)" />
+        <PackageReference Update="Microsoft.Internal.VisualStudio.Shell.Framework" Version="$(VSFrameworkVersion)" />
+
         <PackageReference Update="Microsoft.VisualStudio.Shell.Immutable.15.0" Version="15.0.25123-Dev15Preview" />
-        <PackageReference Update="Microsoft.VisualStudio.Shell.Interop.16.0.DesignTime" Version="$(VSFrameworkVersion)" />
-        <PackageReference Update="Microsoft.VisualStudio.Shell.Interop.16.3.DesignTime" Version="$(VSFrameworkVersion)" />
-        <PackageReference Update="Microsoft.VisualStudio.Shell.Interop.16.6.DesignTime" Version="$(VSFrameworkVersion)" />
-        <PackageReference Update="Microsoft.VisualStudio.Text.Data" Version="16.9.119-preview" />
-        <PackageReference Update="Microsoft.VisualStudio.Text.Logic" Version="16.9.119-preview" />
-        <PackageReference Update="Microsoft.VisualStudio.Text.UI" Version="16.9.119-preview" />
-        <PackageReference Update="Microsoft.VisualStudio.Text.UI.Wpf" Version="16.9.119-preview" />
-        <PackageReference Update="Microsoft.VisualStudio.TextManager.Interop.10.0" Version="10.0.30319" />
-        <PackageReference Update="Microsoft.VisualStudio.Threading" Version="16.10.53-alpha" />
+        <PackageReference Update="Microsoft.VisualStudio.OLE.Interop" Version="17.0.0-preview-2-31221-277" ExcludeAssets="all" PrivateAssets="all" />
+        <PackageReference Update="Microsoft.VisualStudio.Shell.Interop" Version="17.0.0-preview-1-31108-209" ExcludeAssets="all" PrivateAssets="all" NoWarn="NU1605" />
+        <PackageReference Update="Microsoft.VisualStudio.Shell.Interop.8.0" Version="17.0.0-preview-1-31108-209" ExcludeAssets="all" PrivateAssets="all" NoWarn="NU1605" />
+        <PackageReference Update="Microsoft.VisualStudio.Interop" Version="$(VSFrameworkVersion)" NoWarn="NU1605" />
+        <PackageReference Update="Microsoft.VisualStudio.Interop.Embeddable" Version="$(VSFrameworkVersion)" />
+        <PackageReference Update="Microsoft.Internal.VisualStudio.Interop" Version="$(VSFrameworkVersion)" NoWarn="NU1605" />
+        <PackageReference Update="Microsoft.Internal.VisualStudio.Interop.Embeddable" Version="$(VSFrameworkVersion)" />
+        <PackageReference Update="Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime" Version="$(VSFrameworkVersion)" />
+        <PackageReference Update="Microsoft.VisualStudio.TemplateWizardInterface" Version="$(VSFrameworkVersion)" />
+        <PackageReference Update="Microsoft.VisualStudio.Text.Data" Version="17.0.24-g4265d215ae" NoWarn="NU1605" />
```

Comment:
`PrivateAssets` is redundant when `ExcludeAssets` is used. But similar to what Nikolche wrote about the Microsoft.Build.Runtime package, we generally prefer putting NoWarn and Exclude/Private Assets in the project file, rather than packages.targets.